hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/TransactionDispatcher.java

```diff
@@ -329,12 +331,25 @@ protected void finishConsensusSubmitMessage(
      * @param tokenRelStore the token relation store
      */
     private void dispatchTokenGrantKycToAccount(
-            TransactionBody tokenGrantKyc, WritableTokenRelationStore tokenRelStore) {
+            @NonNull final TransactionBody tokenGrantKyc, @NonNull final WritableTokenRelationStore tokenRelStore) {
         final var handler = handlers.tokenGrantKycToAccountHandler();
         handler.handle(tokenGrantKyc, tokenRelStore);
         tokenRelStore.commit();
     }
 
+    /**
+     * Dispatches the token revoke KYC transaction to the appropriate handler.
+     *
+     * @param tokenRevokeKyc the token revoke KYC transaction
+     * @param tokenRelStore the token relation store
+     */
+    private void dispatchTokenRevokeKycFromAccount(
+            @NonNull TransactionBody tokenRevokeKyc, @NonNull WritableTokenRelationStore tokenRelStore) {
+        final var handler = handlers.tokenRevokeKycFromAccountHandler();
+        handler.handle(tokenRevokeKyc, tokenRelStore);
+        tokenRelStore.commit();
```

Comment:
These `*Store.commit()` calls will be done by the workflow somewhere else eventually, so atm are just a `mono-service`-adapter pattern.

So we might move this call into a `MonoTransactionDispatcher#finishTokenRevokeKycFromAccount()` method a la [here](https://github.com/hashgraph/hedera-services/blob/f80240f84ac872a9590e0eaf286c310509cb1507/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/MonoTransactionDispatcher.java#L61).